.github/CODEOWNERS

```diff
@@ -1443,7 +1443,7 @@
 /types/fill-pdf/                                    @westy92
 /types/filter-console/                              @BendingBender
 /types/filter-invalid-dom-props/                    @icopp
-/types/finalhandler/                                @chrootsu
+/types/finalhandler/                                @pillarjs
```

Comment:
You don't need to edit this file directly; we'll update it in bulk based on your change to the header of `index.d.ts`.

Reply:
Removed this change.